src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -58,6 +58,8 @@ class PhpDumper extends Dumper
     private $targetDirRegex;
     private $targetDirMaxMatches;
     private $docStar;
+    private $serviceIdToMethodNameMap = array();
+    private $usedMethodNames = array();
```

Comment:
This seems complicated to do as the base container class can be changed in the ``dump()`` options.